Language-Version-History.md

```diff
@@ -111,11 +112,23 @@ Features Added in C# Language Versions
 # [C# 1.0](https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#.NET_.282002.29) - Visual Studio .NET 2002
 - Classes
 - Structs
+- Enums
 - Interfaces
 - Events
+- [Operator overloading](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading)
+- [User-defined conversion operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/user-defined-conversion-operators)
 - Properties
+- [Indexers](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/indexers/)
+- Output parameters (out and ref)
+- [Array trailing parameter (params)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params)
 - Delegates
 - Expressions
-- Statements
+- [Statements](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/statements)
+- [using statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#the-using-statement)
```

Comment:
Doesn't seem like we need these specific statements, or we should remove the `Statement` bullet entirely.

Reply:
I'd recommend removing the "Statement" bullet.

Reply:
> I'd recommend removing the "Statement" bullet.

Agree!